skeleton/.buildkite/rebuild.hs

```diff
@@ -0,0 +1,69 @@
+{-# LANGUAGE LambdaCase          #-}
```

Comment:
Can we put this somewhere common or does it have to be copied into the project?

Reply:
Somewhere common would be good, although the build steps tend to be slightly different for each project.

There used to be more stuff in this file because it was restoring and saving `.stack-work` directories with `cache-s3`. But the cache transfers are too slow on packet.net, so we removed it.